app/services/itunesApi.js

```diff
@@ -0,0 +1,4 @@
+import { generateApiClient } from '@utils/apiUtils';
+const itunesApi = generateApiClient('itunes');
```

Comment:
A constant for `itunes` is already defined in the `app/utils/apiUtils.js` file. If a constant is already present for a string please use that instead of hardcoding the string directly so that in future if you decide to update the string value then you only need to do it in one place.

Reply:
API type string imported from enum.